class.jetpack-connection-banner.php

```diff
@@ -873,7 +873,7 @@ class="jetpack-logo"
 					?></p>
 					<p><?php
 						esc_html_e(
-							'Optimize: Upload images and videos directly to our powerful servers that present your media with lightning speed.',
+							"Optimize: Jetpack's site accelerator uploads images, videos, and static files like CSS to our powerful servers and delivers them at lightning-fast speeds to your visitors. You'll also save precious bandwidth.",
```

Comment:
Minor nitpick, but we could still use the single quote marks if we keep the correct apostrophe character `’` (vs `'`). I know this is semi-urgent, so feel free to ignore this feedback.

Reply:
How do you actually do that `’`? :)

More seriously, is that a valid English character? I had it on my French keyboard back in the day, but it is not available on my English keyboard now.

Reply:
In this specific case, you could copy it from https://github.com/Automattic/jetpack/pull/10490/files#diff-511f6341fbb24bcadaae3ada1e3c3039L917 but I'm sure it's different for multiple combinations of keyboard layouts and configurations. And it should be a valid character.

Reply:
Done in 960aace3697179dacc10eb79c8daeb1c34f3d35b